<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportMovingDispClients</class>
 <widget class="QDialog" name="ReportMovingDispClients">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="cmbRowGrouping">
     <item>
      <property name="text">
       <string>Подразделениям</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Специальности</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Врачам</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson">
     <item>
      <property name="text">
       <string>Врач</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblRowGrouping">
     <property name="text">
      <string>Группировать по</string>
     </property>
     <property name="buddy">
      <cstring>cmbRowGrouping</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbRowGrouping</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportMovingDispClients</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportMovingDispClients</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
